src/retrieve-w3c-validation/index.ts

```diff
@@ -0,0 +1,15 @@
+// Imports
+import { W3CCSSValidatorResponse } from '..';
+import retrieveForBrowser from './browser';
+import retrieveForNode from './node';
+
+// Function that detects the appropriate HTTP request client and returns a response accordingly
+const retrieveW3CValidation = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```

Comment:
Let's simplify this to just:

```suggestion
const retrieveValidation = async (url: string): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
```

The entire package is dedicated to using W3C's validator, so we don't need to repeat that term in the function names and file names. Speaking of, this will need to be moved from `retrieve-w3c-validation/` to `retrieve-validation/`.